src/main/resources/application.properties

```diff
@@ -3,4 +3,4 @@ spring.datasource.url=jdbc:mysql://localhost:3306/admin?serverTimezone=UTC
 spring.datasource.username=root
 spring.datasource.password=admin
 
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
```

Comment:
why you deleted and next added the same?

Reply:
This line was modified I only added new line at the end